tests/test_base.py

```diff
@@ -29,6 +29,10 @@ def _test_fit_transform(self, tw_model, expected, atol=1e-6):
         predicted = tw_model.fit_transform(docs)
         # torch.set_printoptions(precision=10)
         # print(predicted)
+        if not torch.allclose(predicted, expected.to(device), atol=atol):
```

Comment:
do we want the prints()? And can you add a one-liner comment on this in the code about what it does please?

Reply:
@skadio These lines existed before, and I don't think they are necessary.

Also, this is still WIP, so please bare with me, I put a label and didn't tag you for the same reason. I will get back once, I solve the test failures.

Reply:
ah ok.. somehow it asked me for "Review". no worries